src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/ContainerProperties.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.List;
+
+@ConfigurationProperties(prefix = "containers")
+public class ContainerProperties {
```

Comment:
as those are not properfies of a container, a more clear name or comment on a class would be nice.

Reply:
I renamed it as `ContainerMappings`